src/actions/PostActionCreators.js

```diff
@@ -5,33 +5,37 @@ var sectionIdsCursor = store.select('models', 'sectionIds');
 var { rawDataToPost } = require('../utils/Post');
 
 var urlBuilder = (sectionName) => {
+    if (sectionName !== 'frontpage') {
```

Comment:
Please avoid re-indenting entire files in your editor. Also, use two spaces, not four.